packages/cfd/src/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -107,21 +105,8 @@ const CFDFinancialStpRealAccountSignup = observer(({ onFinish }: TCFDFinancialSt
 
     const poa_config: TItemsState<typeof passthroughProps> = {
         body: CFDPOA,
-        form_value: {
-            address_line_1: account_settings.address_line_1,
-            address_line_2: account_settings.address_line_2,
-            address_city: account_settings.address_city,
-            address_state: account_settings.address_state,
-            address_postcode: account_settings.address_postcode,
-            upload_file: '',
-        },
-        forwarded_props: [
-            'states_list',
-            'account_settings',
-            'storeProofOfAddress',
-            'refreshNotifications',
-            'jurisdiction_selected_shortcode',
-        ],
+        form_value: {},
```

Comment:
why are we removing default values?

Reply:
they are not passed to cfd-poa component via props. all needed fields we are getting there with useStore